src/coiio/imageio.cpp

```diff
@@ -0,0 +1,201 @@
+#include <OpenImageIO/imageio.h>
+
+#include "imageio_defines.h"
+#include "typedesc.h"
+#include "util.hpp"
+
+using ImageInput  = OIIO::ImageInput;
+using ImageOutput = OIIO::ImageOutput;
+using ImageSpec   = OIIO::ImageSpec;
+
+extern "C" {
+
+ImageSpec*
+ImageSpec_new()
+{
+    return new ImageSpec;
+}
+
+void
+ImageSpec_delete(const ImageSpec* is)
+{
+    delete is;
+}
+
```

Comment:
No apology necessary! I'll fix this. Incidentally, what version of clang-format are you using? Mine (9.0.0) barfs on the SpaceInEmptyBlock line.

Reply:
Yeah, you need to use clang10 for clang-format, I guess. FWIW, clang-format 11 fixes this -- ignoring unknown options instead of considering them errors.